p/scripts/config.js

```diff
@@ -0,0 +1,48 @@
+// @license magnet:?xt=urn:btih:0b31508aeb0634b347b8270c7bee4d411b5d4109&dn=agpl-3.0.txt AGPL-3.0
+'use strict';
+
+function init_selectInputChanger() {
+	const select = document.getElementsByClassName('select-input-changer');
+
+	for (let i = 0; i < select.length; i++) {
+		select[i].addEventListener('click', function () { onchange_selectInputChanger(this); });
+		onchange_selectInputChanger(select[i]);
+	}
+}
+
+function onchange_selectInputChanger(elem) {
```

Comment:
I would name this something more generic, otherwise it's very easy to get at best strange and at worst misleading function names like we see here. However, the function name is correct that you should probably be listening for change, not for click.
```suggestion
function updateSelectInput(elem) {
```